modules/ossm-tutorial-bookinfo-verify-install.adoc

```diff
@@ -10,17 +10,10 @@ Before configuring your application, verify that it successfully deployed.
 
 .Procedure
 
-* To confirm that the application is deployed:
-
-** Run this command:
+* Run this command to confirm that Bookinfo is deployed:
 +
 ----
 $ curl -o /dev/null -s -w "%{http_code}\n" http://<GATEWAY_URL>/productpage
```

Comment:
Should this be something simpler like:

Run this command to confirm that Bookinfo is deployed:

`$ oc get pods -n bookinfo`

Verify that all pods are ready.

Reply:
The problem with doing this is you're only making sure the pod is running. Not that it's actually running, attached to the mesh, and routable via the Istio ingress gateway.

Reply:
http://<GATEWAY_URL>/productpage is not a valid URL and the curl command will fail.

The command should read:
$ curl -o /dev/null -s -w "%{http_code}\n" http://$GATEWAY_URL/productpage

Also, the bulleted list item:
  Alternatively, you can open http://<GATEWAY_URL>/productpage in your browser.
should be
  Alternatively, you can open http://$GATEWAY_URL/productpage in your browser.

Reply:
Updated this with the @travisrogers05 changes.